src/pyspextool/combine/combine_spectra.py

```diff
@@ -253,9 +253,29 @@ def combine_spectra(statistic:str='robust weighted mean',
                           'Combination statistic']
 
     avehdr['CMB_THSH'] = [robust_sigma,
-                         'Combination robust threshold (if used)']            
+                         'Combination robust threshold (if used)']   
 
+    # Add the S/N ratio 
+
+    for i in range(combine.state['norders']):
```

Comment:
```suggestion
    for order in range(combine.state['norders']):
```
Doesn't need to be for this PR, but it's good to avoid using `ijk` counters and to try to use something more descriptive. Like in this case, if it's looping over the orders, use `order` instead of `i`.

Reply:
Ah ok.  Will try and do that in the future.